flink-connectors/flink-connector-hive/src/main/resources/META-INF/NOTICE

```diff
@@ -6,36 +6,32 @@ The Apache Software Foundation (http://www.apache.org/).
 
 This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt)
 
-- org.apache.hadoop:hadoop-common:2.7.2
 - commons-io:commons-io:2.4
+- commons-io:commons-dbcp:1.4
+- commons-io:commons-pool:1.5.4
 - commons-logging:commons-logging:1.1.3
 - commons-configuration:commons-configuration:1.7
 - commons-beanutils:commons-beanutils:1.8.3
+- com.jolbox:bonecp:0.8.0.RELEASE
 - org.apache.hadoop:hadoop-auth:2.7.2
+- org.apache.hadoop:hadoop-common:2.7.2
 - org.apache.hadoop:hadoop-mapreduce-client-core:2.7.2
+- org.apache.hive:hive-common:2.3.4
+- org.apache.hive:hive-exec:2.3.4
 - org.apache.hive:hive-metastore:2.3.4
 - org.apache.hive:hive-serde:2.3.4
-- org.apache.hive:hive-common:2.3.4
-- org.apache.hive:hive-storage-api:2.4.0
 - org.apache.hive:hive-service-rpc:2.3.4
-- com.jolbox:bonecp:0.8.0.RELEASE
-- org.apache.derby:derby:10.10.2.0
+- org.apache.hive:hive-storage-api:2.4.0
+- org.apache.thrift:libthrift:0.9.3
 - org.datanucleus:datanucleus-api-jdo:4.2.4
 - org.datanucleus:datanucleus-core:4.1.17
 - org.datanucleus:datanucleus-rdbms:4.1.19
-- commons-io:commons-pool:2.4
-- commons-io:commons-dbcp:2.4
 - org.datanucleus:javax.jdo:3.2.0-m3
-- org.apache.thrift:libthrift:0.9.3
-- org.apache.httpcomponents:httpclient:4.5.3
-- org.apache.httpcomponents:httpcore:4.4.6
 
-This project bundles the following dependencies under the CDDL and GPL 2.0 license.
-See bundled license files for details.
+This project bundles the following dependencies under the CDDL and GPL 2.0 license. (https://javaee.github.io/glassfish/LICENSE)
```

Comment:
please bundle the license like we do in other modules; the license should be accessible even if the URL no longer is.